### PR TITLE
fix: show file names for grep command on Linux

### LINKED ISSUE #53: [bat error]: invalid digit found in string
Hey,

I just tried out `fzf-make` on [Shuttle's Makefile](https://github.com/shuttle-hq/shuttle/blob/main/Makefile) and got this error.

cap

Also, I'm getting a different error for the last entry:

cap

Thanks for this tool!

### PR DESCRIPTION
**Pull request by orhun**
fixes #53

### CODE REVIEW

File: src/fuzzy_finder/fuzzy_finder.rs [hunk 1]
```diff
@@ -30,7 +30,9 @@ pub fn run(makefile: Makefile) {
     }
 }
 
-fn get_preview_command(file_paths: Vec<String>) -> String {
+fn get_preview_command(mut file_paths: Vec<String>) -> String {
+    // workaround for https://stackoverflow.com/questions/15432156/display-filename-before-matching-line
```
Comment (kyu08) [inline src/fuzzy_finder/fuzzy_finder.rs:34]: I think the comments you wrote here (https://github.com/kyu08/fzf-make/issues/53#issuecomment-1684872018) will be of great help to future readers of this part (myself included).
So how about adding that link to the code comments like below?

```suggestion
    // workaround for https://stackoverflow.com/questions/15432156/display-filename-before-matching-line
    // For more information, see https://github.com/kyu08/fzf-make/issues/53#issuecomment-1684872018
```

**Review by kyu08 — COMMENTED** [review]
Thanks so much for investigating and fixing this.
I am very grateful that fzf-make now works on Linux thanks to you.
I have confirmed that it works fine on macOS as well.
I just have one request for you to fix it.

File: src/fuzzy_finder/fuzzy_finder.rs (hunk 1 above)
Comment (orhun) [reply src/fuzzy_finder/fuzzy_finder.rs:34]: Sounds reasonable!

**Review by kyu08 — APPROVED** [review]
Thank you for fixing so quickly!

**Comment by kyu08** [comment]
@orhun 
Again, thanks for fixing this issue.
Not only that, but I think the number of fzf-make users has increased since you mentioned fzf-make on twitter.
I am truly grateful to you in this regard as well.

**Comment by orhun** [comment]
Thank you for the merge! 🚀 

Do you mind creating a new release soon? 👁️

**Comment by kyu08** [comment]
@orhun 
v0.7.0 was released! 🚀 
Thank you for contribution!


one question.
How did you install fzf-make?
I want to know how can linux users use fzf-make.

**Comment by orhun** [comment]
Yay! 🥳 

> How did you install fzf-make?

I packaged it in the AUR and installed from there 🐻 

- https://aur.archlinux.org/packages/fzf-make

I can add instructions to README.md if you want 👀

**Comment by kyu08** [comment]
> I packaged it in the AUR and installed from there 🐻

Oh! I see. Thank you for adding fzf-make to AUR!

> I can add instructions to README.md if you want 👀

Very helpful! ❤️ 
Can you give me some time to make a few changes to the README format?

**Comment by kyu08** [comment]
@orhun 
I'm done!

Could you add a how to install here?
https://github.com/kyu08/fzf-make/blob/d82e4aa9b1fd5feba487491f82cf3cf7ba10f631/README.md#L34

**Pull request merged**